### PR TITLE
Replace static readonly with const

### LINKED ISSUE #10: Consider switching to const where applicable
The codebase uses `readonly` most places, but `const` should be fine for literals.

### PR DESCRIPTION
**Pull request by hantatsang**
Closes #10

### CODE REVIEW

File: Yoakke.Lsp.Generator/Program.cs [hunk 1]
```diff
@@ -62,13 +61,13 @@ static void Translate(DefBase def)
...
             switch (def)
             {
-            case InterfaceDef i: 
-                Translate(i); 
-                break;
-            case NamespaceDef n: 
-                Translate(n); 
-                break;
-            default: throw new InvalidOperationException();
+                case InterfaceDef i:
```
Comment (LPeter1997) [inline Yoakke.Lsp.Generator/Program.cs:64]: I would suggest reverting this indentation. We might need to consider switching to this style as it's more traditional C#. For now, I'd recommend sticking with the projects style for consistency.

File: Yoakke.Ast.Generator/TypeNames.cs [hunk 2]
```diff
@@ -1,30 +1,26 @@
...
+        private const string IEnumerable = "System.Collections.Generic.IEnumerable";
+        private const string StringBuilder = "System.Text.StringBuilder";
+        private const string KeyValuePair = "System.Collections.Generic.KeyValuePair";
+        private const string IReadOnlyCollection = "System.Collections.Generic.IReadOnlyCollection";
         // Loading
-        public static readonly string IReadOnlyListG = "System.Collections.Generic.IReadOnlyList`1";
-        public static readonly string IReadOnlyCollectionG = "System.Collections.Generic.IReadOnlyCollection`1";
+        private const string IReadOnlyListG = "System.Collections.Generic.IReadOnlyList`1";
+        private const string IReadOnlyCollectionG = "System.Collections.Generic.IReadOnlyCollection`1";
     }
```
Comment (LPeter1997) [inline Yoakke.Ast.Generator/TypeNames.cs:25]: Sadly, private won't work here, please refer to the build

**Review by LPeter1997 — CHANGES_REQUESTED** [review]

File: Yoakke.Lsp.Generator/Program.cs (hunk 1 above)
Comment (hantatsang) [reply Yoakke.Lsp.Generator/Program.cs:64]: Hmm... this accidental change is automated by my IDE, will revert

**Comment by LPeter1997** [comment]
Thank you very much for your contribution!

It looks like the CI pipeline is still not 100% great with "outsider" PRs as the secrets (tokens) are not shared properly. Sadly I don't know if forked PRs are even able to see secrets as of right now.

**Pull request merged**